scripts/format

```diff
@@ -13,7 +13,12 @@ find oak examples toolchain \( -type f -name BUILD -or -name WORKSPACE -or -name
   -exec buildifier -lint=fix -mode=fix {} +
 find oak examples \( -type f -name '*.h' -or -name '*.cc' -or -name '*.proto' \) \
   -exec clang-format -i -style=file {} +
-find examples rust -type f -name '*.rs' -exec rustfmt {} +
+find examples rust \
+  \(  \
+    -not \( -path examples/target -prune \) -and \
```

Comment:
I feel this is gradually getting out of hand, perhaps we should try and write a Bazel rule that does this at some point. Not sure it would work given it would overlap with other bazel targets though. Let me know if you have any ideas.

Reply:
A Bazel rule would make sense if we were building Rust code with it, but we're not.  That did make me think though: we're building Rust code with `cargo`, so maybe it can take care of filtering just for local code&hellip;